plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
Agreed it's probably impossible to account for all quirks and issues with every possible version of all drivers.

But I still think we should use something with higher resolution than vendor only. Even if it won't be perfect, and it won't be, it'll reduce the false assumptions by a considerable amount IMHO.